docs/api-aggregation-setup.md

```diff
@@ -177,6 +177,10 @@ configured name ${HELM_NAME}, and into the specified namespace." This
 command also enables authentication and aggregation and provides the
 keys we just generated inline.
 
+The installation commands vary slightly between Linux and Mac OS X because of
```

Comment:
I believe the official name has changed to from "Mac OS X" to "macOS". :sweat_smile: